### DISCUSSION TITLE
unable to play yt video thru google

### DISCUSSION THREAD

**Discussion by suicideboy98** [bromite/bromite#1421] [Q&A]
it works in chrome and brave 
link example
![image](https://user-images.githubusercontent.com/50690339/134805218-5209cd66-1d77-43ed-b249-07438960719e.png)
- https://www.google.com/search?q=bullfrog+exclusive+ace&oq=bullfrog+&aqs=chrome.2.69i57j0i512j69i59j0i512l5j46i199i465i512j0i512.4002j0j9&sourceid=chrome-mobile&ie=UTF-8#fpstate=ive&vld=cid:ee2fb018,vid:ej8BbIGZpa4,st:146

**Comment by Oymate** [comment]
Use this link: https://m.youtube.com/watch?v=ej8BbIGZpa4

**Comment by Oymate** [comment]
Press the arrow link to open the video through google. This is probably related to 3rd party shenanigans, expected behavior.
